packages/cf-deploy-config-inquirer/src/types.ts

```diff
@@ -71,10 +112,50 @@ export interface CfDeployConfigAnswers {
     destinationName?: string;
     /** Indicates whether the user opted to include a managed application router. */
     addManagedRouter?: boolean;
-    /* Indicates whether the user opted to overwrite the destination. */
+    /** Indicates whether the user opted to overwrite the destination. */
     overwrite?: boolean;
 }
 
+/**
+ * Enum defining the types of router modules for the Application Router configuration.
+ */
+export enum RouterModuleType {
```

Comment:
Can you ensure this type is exported at the module level as the value is required for telemetry?

Reply:
Better to not use enums when exporting types to avoid dependencies (instead of dev dependencies).

Reply:
For example;
```JS
export const RouterModuleType = {
 Standard = 'standard',
 Managed = 'managed'
} as const;

export type RouterModuleType = typeof RouterModuleType[keyof typeof RouterModuleType];
```

Reply:
Thanks - changed [RouterModuleType](https://github.com/SAP/open-ux-tools/pull/2490/files#diff-24301133b22fc485c7fb65d11b3e76c2150400123d73802940fe378e8b378ebdR122), also exported to be used by telemetry...